test/functional/admin/api/cms/templates_controller_test.rb

```diff
@@ -93,7 +93,7 @@ def test_create_page_section_name
       end
 
       def test_create_page_section_id_unknown
-        post :create, params: { section_id: 100, type: 'page',
+        post :create, params: { section_id: 1000000, type: 'page',
```

Comment:
Hmmm, I am not sure I understand... Is it that sometimes factory-created sections may get `100` as ID, but not 1000000?...

Reply:
Exactly. When I wrote the tests initially I though 100 was a number high enough to ensure will never be a section with that id during test time. I was wrong.

Reply:
I have a slight urge to extract the 1000000 value to a constant, smth like `NON_EXISTING_ID`, just to avoid repeating the same number over again (and thinking - why this number?).

But not sure whether it's actually a good or a bad idea.

Reply:
Done!